generic/tests/cfg/lvm.cfg

```diff
@@ -11,7 +11,8 @@
     image_size_stg2 = 1G
     image_format_stg2 = qcow2
     guest_testdir = /mnt
-    kill_vm = no
+    mount_lv = yes
```

Comment:
For your current change, you could remove `if mount_lv == "yes":` and rewrite log message.